core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/ProvisioningContext.java

```diff
@@ -416,7 +434,7 @@ protected Transport connectTransport() throws MessagingException {
             if (provisioningProperties.getSmtp().isDebug()) {
                 session = mailSender.getSession();
                 session.setDebug(true);
-                try (LogOutputStream los = new LogOutputStream(LOG)) {
+                try ( LogOutputStream los = new LogOutputStream(LOG)) {
```

Comment:
Please revert this formatting-only change.
It will also break checkstyle AFAICT.